samples/drivers/can/counter/boards/imx93_evk_mimx9352_a55.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_I2C=y
+CONFIG_MFD=y
```

Comment:
why CONFIG_GPIO is not enabled at this case but enabled with all the other case?

Reply:
it was enabled in prj.conf in this particular project.